src/Compilers/VisualBasic/Test/Emit/Emit/EditAndContinueTests.vb

```diff
@@ -3325,7 +3326,7 @@ End Namespace
                                                                      testData0.GetMethodData("M.B.M").EncDebugInfoProvider)
                 Dim method0 = compilation0.GetMember(Of MethodSymbol)("M.B.M")
                 Dim reader0 = md0.MetadataReader
-                CheckNames(reader0, reader0.GetTypeDefNames(), "<Module>", "VB$AnonymousType_1`2", "VB$AnonymousType_0`2", "VB$AnonymousType_2`1", "A", "B")
+                CheckNames(reader0, reader0.GetTypeDefNames(), "<Module>", "VB$AnonymousType_1`2", "VB$AnonymousType_2`1", "VB$AnonymousType_0`2", "A", "B")
```

Comment:
Unless you fixed the compiler to deterministically enumerate anonymous types this will break. Currently the compiler enumerates a dictionary, so the order depends on the has code algorithm used by the dictionary, which is presumably different between x86 and x64. There is an issue filed to address this problem.